content/user/email/all_customers.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How do I email all my customers about a sale or promotion? 
+description: Sending bulk email 
+category: email
+weight: 10
+---
+
+Sending large numbers of emails from your own hosting account 
+is no longer recommended.  
```

Comment:
Perhaps "no longer" would be better said as "not". People reading this won't know that there ever "was" any recommendation to do so (and really there wasn't any recommendation, just a "want", because "free").

Also, why such short lines? line-breaks in middle of thoughts?

Reply:
changed to "not" and fixed line breaks.